zerver/views/registration.py

```diff
@@ -268,18 +270,22 @@ def accounts_register(request):
                  }
     )
 
-def create_preregistration_user(email, request, realm_creation=False):
-    # type: (Text, HttpRequest, bool) -> HttpResponse
+def create_preregistration_user(
+        email, request, realm_creation=False, is_remote=False):
```

Comment:
we generally wrap functions with the first few arguments still on the first line.